bin/terminus.php

```diff
@@ -30,6 +30,12 @@
 $output = new ConsoleOutput();
 Robo::configureContainer($container, $input, $output, $application);
 
+$container->share('fileCache', \Terminus\Caches\FileCache::class);
+$container->share('session', \Pantheon\Terminus\Session\Session::class)
+    ->withArgument('fileCache');
+$container->inflector(\Pantheon\Terminus\Session\SessionAwareInterface::class)
```

Comment:
Looks like it's because Robo adds inflectors for those automatically: https://github.com/consolidation-org/Robo/blob/d712e16b1dea2a218434a6654277ddce945a10a6/src/Robo.php#L159